packages/lab/src/cascading-menu/CascadingMenu.tsx

```diff
@@ -195,9 +194,7 @@ export const CascadingMenu = forwardRef<HTMLDivElement, CascadingMenuProps>(
       childrenRef.current = node;
     }, []);
     const handleRef = useForkRef(
-      //TODO get to bottom of this
-      // @ts-ignore
-      isValidElement(children) ? children.ref : noop,
```

Comment:
This one returned `noop` not `null` like in the other usages.  

The update looks Ok but just pointing out the diff

Reply:
Yeah, i think this was just an old way of doing it